packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -0,0 +1,249 @@
+};
+
+/**
+ * Use the 'base', 'summary-container', 'content', and 'button' parts to style the sd-quickfact component.
+ */
+export const Parts = {
+  parameters: {
+    controls: {
+      exclude: ['base', 'summary-container', 'content', 'button']
+    }
+  },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'template',
+          name: 'sd-quickfact::part(...){outline: solid 2px red}',
+          values: ['base', 'summary-container', 'content', 'button'].map(part => {
+            return {
+              title: part,
+              value: `<style>#part-${part} sd-quickfact::part(${part}){outline: solid 2px red; outline-offset: -2px;}</style><div id="part-${part}">%TEMPLATE%</div>`
+            };
+          })
+        }
+      },
+      args,
+      constants: [
+        {
+          type: 'attribute',
+          name: 'open',
+          value: true
+        }
+      ]
+    });
+  }
+};
+
+/**
+ * Use the `default`, `icon`, `summary`, `expand-icon` and `collapse-icon` slots to add content to the step..
+ */
+
+export const Slots = {
+  parameters: {
+    controls: { exclude: ['default', 'icon', 'summary', 'expand-icon', 'collapse-icon'] }
+  },
+  render: (args: any) => {
+    return html`
+      ${['default', 'icon', 'summary', 'expand-icon', 'collapse-icon'].map(slot =>
+        generateTemplate({
+          axis: {
+            y: {
+              type: 'slot',
+              name: slot,
+              title: 'slot=..',
+              values: [
+                {
+                  value:
+                    slot === 'default'
+                      ? `<div class="slot slot--border slot--background h-12"></div>`
+                      : `<div slot='${slot}' class="slot slot--border slot--background min-w-6 h-12"></div>`,
+                  title: slot
+                }
+              ]
+            }
+          },
+          args,
+          constants: [
+            {
+              type: 'attribute',
+              name: 'open',
+              value: true
+            }
+          ]
+        })
+      )}
+    `;
+  }
+};
+
+/**
+ * sd-quickfacts are fully accessibile via keyboard.
+ */
+
+export const Mouseless = {
+  render: (args: any) => {
+    return html`<div class="mouseless">${generateTemplate({ args })}</div>`;
+  },
+
+  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
+    const el = canvasElement.querySelector('.mouseless sd-quickfact');
+    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
+    // We have to catch the event as otherwise Storybook will break
+    await userEvent.type(el!.shadowRoot!.querySelector('button')!, '{return}', { pointerEventsCheck: 0 });
+  }
+};
+
+/**
+ * This sample shows how to group sd-quickfacts together. It also includes a script that closes all other quickfacts when one is opened. Additionally, it uses the `content` CSS part to position the default slot to cover the entire width of the group.
```

Comment:
I would like to see the default and optional content being represented here. From what I read in our teams chat, @coraliefeil mentioned to have the "headline" as "required" but the summary optional.
As we cannot define a required slot content, I would like to have this displayed in this story. 
In addition to this, we (IMO) need to show the expected design behavior here in case the summary content has a different hight. Even though this is not a feature of the component and relating to @EllaELSA `s comment in the teams chat, its a design requirement and this should be displayed in Figma and Storybook.

@mariohamann @yoezlem FYI

Reply:
The naming "summary" is related to the naming in `details`/`summary`, as this is actually an accordion in a different look. We only have one slot for the area at the top of the arrow. I think we should just give it a default styling (`text-base font-normal leading-normal  sm:text-3xl sm:leading-tight`) and everything else could be handled in the slot, like the "sub-headline".

Reply:
my concern is not about the technical solution but more on the old topic providing flexibility but also guidance on how to handle the flexibility.